rust/VENDOR.md

```diff
@@ -0,0 +1,51 @@
+As we have multiple projects we use a workspace to manage them (it's way simpler and leads to less issues). In order
```

Comment:
A very cool thing is done here!

Does adding a script into https://github.com/ClickHouse/rust_vendor to update everything as a simple command make sense?

Reply:
Maybe.

I generated the commands matching exactly how `std` is organised and our current `Cargo.lock` so they might not be universal. We can wait to execute them a few times in the future (understanding the commands) before making it automatic.